cli/src/cmd/forge/script.rs

```diff
@@ -79,6 +78,18 @@ pub struct ScriptArgs {
 
     #[clap(flatten, next_help_heading = "EVM OPTIONS")]
     pub evm_opts: EvmArgs,
+
+    #[clap(
+        long,
+        help = "resumes previous transaction batch. does NOT simulate execution. respects nonce constraint "
+    )]
+    pub resume: bool,
+
+    #[clap(
+        long,
+        help = "resumes previous transactions batch. does NOT simulate execution. does not respect nonce constraint"
```

Comment:
Can you emphasize the "does not" here, maybe caps it? i had to read this a couple times before i actually noticed the difference and almost made this comment be "can you document the difference?".